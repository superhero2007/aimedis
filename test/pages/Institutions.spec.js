import React from 'react';
import { shallow } from 'enzyme';

import Institutions from 'pages/Institutions';

function setup() {
  return shallow(<Institutions />);
}

describe('Institutions', () => {
  const wrapper = setup();

  it('should be a Component', () => {
    expect(wrapper.instance() instanceof React.Component).toBe(true);
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
