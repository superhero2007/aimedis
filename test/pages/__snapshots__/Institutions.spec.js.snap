// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Institutions should render properly 1`] = `"<div class=\\"app__institutions app__route\\"><div class=\\"container\\"><section class=\\"app__institutions__section1\\"><div class=\\"app__institutions__section1__title\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">institutions</p><p class=\\"app__title__title\\">Who gets the most out of Aimedis</p></div></div><div class=\\"app__institutions__section1__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-5\\"><div class=\\"app__overview active\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">1</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Insurance companies</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">Excepteur sint accaecut cupidatat n every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div><button class=\\"app__overview__external__download body btn btn__secondary ml-0\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">2</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Hospitals</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">Excepteur sint accaecut cupidatat n every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div><button class=\\"app__overview__external__download body btn btn__secondary ml-0\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div><div class=\\"app__overview\\"><div class=\\"app__overview__internal\\"><div class=\\"app__overview__internal__content\\"><div class=\\"app__overview__internal__content__order\\"><p class=\\"app__overview__internal__content__order__text body\\">3</p></div><div class=\\"app__overview__internal__content__paragraph\\"><p class=\\"app__overview__internal__content__paragraph__title title\\">Pharma Companies</p><p class=\\"app__overview__internal__content__paragraph__subtitle body\\">Excepteur sint accaecut cupidatat n every actual browser</p></div><div class=\\"app__overview__internal__content__next ml-auto\\"><span class=\\"i-long-arrow-right d-none d-lg-inline\\"></span><span class=\\"i-long-arrow-down d-inline d-lg-none\\"></span></div></div></div><div class=\\"app__overview__external\\"><div class=\\"app__overview__external__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div><button class=\\"app__overview__external__download body btn btn__secondary ml-0\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div></div><div class=\\"col-lg-7 d-none d-lg-block app__institutions__section1__content__image\\"><img src=\\"\\" alt=\\"No Image\\"/></div></div></div></section><section class=\\"app__institutions__section2 d-none d-lg-block\\"><div class=\\"app__institutions__item\\"><div class=\\"app__institutions__item__content\\"><div class=\\"row\\"><div class=\\"col-6 app__institutions__item__content__info\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">INSURANCE COMPANIES</p><p class=\\"app__title__title\\">Dois aute irure dolor</p></div><p class=\\"app__institutions__item__content__info__description\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help.</p><button class=\\"app__institutions__item__content__info__download btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div><div class=\\"col-6 app__institutions__item__content__image\\"> <img src=\\"\\" alt=\\"No Image\\"/> </div></div></div></div><div class=\\"app__institutions__item\\"><div class=\\"app__institutions__item__content\\"><div class=\\"row\\"><div class=\\"col-6 direction-rtl app__institutions__item__content__image\\"> <img src=\\"\\" alt=\\"No Image\\"/> </div><div class=\\"col-6 app__institutions__item__content__info\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Hospitals</p><p class=\\"app__title__title\\">Dois aute irure dolor</p></div><p class=\\"app__institutions__item__content__info__description\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help.</p><button class=\\"app__institutions__item__content__info__download btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div></div></div></div><div class=\\"app__institutions__item\\"><div class=\\"app__institutions__item__content\\"><div class=\\"row\\"><div class=\\"col-6 app__institutions__item__content__info\\"><div class=\\"app__title\\"><p class=\\"app__title__subtitle\\">Pharma Companies</p><p class=\\"app__title__title\\">Dois aute irure dolor</p></div><p class=\\"app__institutions__item__content__info__description\\">Soon Aimedis will offer much more including AIMSocial - the interactive patient platform that is secured and supported by a blockchain and help.</p><button class=\\"app__institutions__item__content__info__download btn btn__secondary\\" type=\\"button\\">DOWNLOAD THE CASE STUDY (8.4 MB)</button></div><div class=\\"col-6 app__institutions__item__content__image\\"> <img src=\\"\\" alt=\\"No Image\\"/> </div></div></div></div></section><section class=\\"app__institutions__section3\\"><div class=\\"app__feature\\"><div class=\\"app__feature__title\\">Discover the best features for you.</div><div class=\\"app__feature__content\\"><div class=\\"app__get__early__access\\"><input type=\\"text\\" class=\\"app__get__early__access__input\\" placeholder=\\"Your Email Address\\"/><button class=\\"app__get__early__access__button btn btn__primary\\">Get <span class=\\"d-none d-lg-inline\\">Early </span>Access</button></div></div></div></section></div></div>"`;
